charts/fluentd/values.yaml

```diff
@@ -46,6 +46,24 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
+# Congigure the livessProbe
+# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+livenessProbeConfig: {}
+  # initialDelaySeconds: 0
+  # periodSeconds: 10
+  # timeoutSeconds: 1
+  # successThreshold: 1
+  # failureThreshold: 3
+
+# Congigure the readinessProbe
+# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+readinessProbeConfig: {}
+  # initialDelaySeconds: 0
+  # periodSeconds: 10
+  # timeoutSeconds: 1
+  # successThreshold: 1
+  # failureThreshold: 3
+
```

Comment:
can you also add startup probe? if it is not too much to ask sorry :) 

I don't really see an use in stand alone, but for the use of this as an aggregator is actually usefull as you might need rolling pod replacement

Reply:
ahh nevermind